ddev/src/ddev/cli/clean/__init__.py

```diff
@@ -0,0 +1,21 @@
+# (C) Datadog, Inc. 2023-present
```

Comment:
What about `ddev/cli/clean.py`? Using __init__ seems like premature optimization in this case.

Reply:
done!